upgrade/helpers/tasks.py

```diff
@@ -923,3 +923,35 @@ def repository_cleanup(repo_name):
     for fname in os.listdir('/etc/yum.repos.d/'):
         if repo_name in fname.lower():
             os.remove('/etc/yum.repos.d/{}'.format(fname))
+
+
+def update_scap_content():
+    """ The purpose of this function is to perform deletion of old scap-contents
+    and then uploading new scap-contents. It also updates scap-policies and assign
+     new scap-content to them """
+    sc = entities.ScapContents().search()
+    sc_id = []
+    cp = entities.CompliancePolicies().search()
+    cp_id = []
+    tmp_sc_id = 1
+    for i in range(len(sc)):
```

Comment:
Please use the variable name more specific to the component.

Reply:
@devendra104 I am just using this to iterate over contents of  `sc`, and I don't know what more specific name I can give it.

Reply:
here you are iterating the number so you can use _ in place of i.

Reply:
much better, will update PR.